identities/src/main/java/org/commonjava/atlas/maven/ident/util/ArtifactPathInfo.java

```diff
@@ -94,7 +101,15 @@ public static ArtifactPathInfo parse( final String path )
         // The classifier can contain dots or hyphens, it is hard to separate it from type. e.g,
         // wildfly8.1.3.jar, project-sources.tar.gz, etc. We don't have a very solid pattern to match the classifier.
         // Here we use the best guess.
-        for ( String type : SPECIAL_TYPES )
+        final String typesFromSys = System.getProperty( COMPOUND_EXTENSIONS_PROP );
+        if ( StringUtils.isNotBlank( typesFromSys ) )
+        {
+            for ( final String type : typesFromSys.split( "," ) )
+            {
+                COMPOUND_EXTENSIONS_TYPES.add( type.trim() );
```

Comment:
`COMPOUND_EXTENSIONS_TYPES` is a final variable, I guess it does not work here to add others.

Reply:
I see. Ignore the comment here. LGTM.

Reply:
Sounds good as static final should be an immutable var. Let's change.